app/views/pickup_checklists/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="content-wrapper-large content-container">
+  <h3>
+    <%= t(".header") %>
+    <%= link_to(
+      zone_scheduled_pickup_path(@pickup_checklist.zone, @pickup_checklist),
+      class: "pickup-checklist-date",
+    ) do %>
+      <%= l(@pickup_checklist.date) %>
+    <% end %>
```

Comment:
This is a little awkward in ERB. Could a helper method improve the display here?

(Also, is there another way to do the path without having to pass two items in? Is there a `pickups` path that can figure out the path just from the checklist without the zone?)